sibylapp/view/feature_contribution.py

```diff
@@ -2,10 +2,31 @@
 
 from sibylapp.compute import contributions
 from sibylapp.compute.context import get_term
+from sibylapp.config import PREDICTION_TYPE, PredType, negativeTerm, positiveTerm
 from sibylapp.view.utils import filtering, helpers
+from sibylapp.view.utils.helpers import NEG_EM, POS_EM
+
+
+def legend():
+    modelPred = get_term("Prediction")
```

Comment:
It looks like the legend is still not showing for the global factor contributions tab under Understand the Model